drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -685,6 +685,29 @@
       end
     end
 
+    it 'handles SSN and DOB fields being hidden' do
```

Comment:
One thing.  Does it make sense to test the cases where the original values are blank and it adds 99 for the DQ values?

Reply:
@eanders I believe this test does test for that. It tests on both a current and a new client, the latter of which has `nil` for all the DQ fields

Reply:
Oh, so the 99 assigned in the client_processor.rb would get eaten and re-converted to a nil?

Reply:
If the value was 99, then it would stay 99. If the value is `nil`, then it would be set to 99, since that column cannot be nil.

Reply:
Nope, I was missing something. Should be good now.